Gruntfile.js

```diff
@@ -100,17 +116,17 @@ module.exports = function (grunt) {
           port: 9001,
           middleware: function (connect) {
             return [
-              connect.static('.tmp'),
+              connect.static('<%= yeoman.stage %>/'),
               connect.static('test'),
               connect().use(
-                '/bower_components',
-                connect.static('./bower_components')
+                '/vendor',
```

Comment:
Are you sure that `grunt connect:test` run minification first? I would personally expect that `connect:test` uses original un-minified files.